technical-reports/color/token-naming.md

```diff
@@ -0,0 +1,377 @@
+# Token naming
+
+## Categorization
+
+There are 3 main categories of design tokens that we will continue to reference in this specification.
+
+### Base
+
+Base tokens are the lowest level tokens and typically consist of a name and [hexadecimal](https://www.w3.org/TR/css-color-4/#hex-notation) value pair.
+
+<aside class="example">
+
+```scss
+$color-green-500: #abcabc;
+$color-shadow-translucent: #00000088;
+```
+
+</aside>
+
+### Alias
+
+A design token’s value can be a _reference_ to another token. The same value can have multiple names or aliases.
+
+<aside class="example">
+
+```scss
+$color-palette-black: #000000;
+$color-text-base: $color-palette-black;
+```
+
+</aside>
+
+### Component
+
+Component-specific tokens provide design decisions at the component level and improve the separation of concerns in your token architecture.
+
+<aside class="example">
+
+```scss
+$color-button-primary: $color-brand-primary;
+$color-banner-background: $color-palette-black;
+```
+
+</aside>
+
+## Naming strategies
+
+There are many naming options when it comes to design tokens, especially color type tokens. We’ve identified two that seem to be most commonly implemented, **descriptive and numerical**.
+
+### Base Tokens
+
+For **Base tokens**, here’s how they may be represented in each version:
+
+#### Descriptive
+
+Descriptive names can be more emotional and human-friendly because they often relate to tangible things that people interact with, like grass or watermelons.
```

Comment:
I would note that using names like "grass" or "canary" may not make sense to someone whose first language is not English.  For projects that require engagement from designers and developers across languages this may not be a suitable choice.

Reply:
Great point, thanks @cgatian!